docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,89 @@
+How to configure boot parameters on Ubuntu Core 
+================================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+Ubuntu Core doesn't have the :code:`/etc/default/grub` file mapped for the kernel parameters, nor the `update-grub`_ utility.
+On Ubuntu Core, the kernel parameters can be modified statically or dynamically.
+Static configurations are added during the image build.
+On the other hand, dynamic configurations are set `using snap options`_ on a running system.
+This guide focuses on the dynamic configuration method, suitable for iteratively tuning the operating system.
```

Comment:
```suggestion

This guide focuses on the dynamic method for configuring kernel boot parameters, suitable for iteratively tuning the operating system.
```

Suggesting a line break and complete context here for readability. You can easily skim the objective of this how-to guide from that one line.

Reply:
![image](https://github.com/user-attachments/assets/aea7cedf-e1c1-4d7b-8233-3c71e3bcdc14)

So it looks like this.